src/runtime/test/strategies/map-slots-tests.ts

```diff
@@ -125,7 +125,9 @@ ${recipeManifest}
 
       recipe
         A
+          consume root 
```

Comment:
why do we need to add these lines now?

Reply:
When slots and slot connections were made eagerly, these slot connections (and provide slots) were created before this strategy was called. The resolveRecipe strategy then called onSlotConnection over the existing slot connections and mapped them to the correct slots. 

Now that slots/slot connections are no longer made eagerly, the resolveRecipe strategy calls onPotentialSlotConnection over the consume slot connections that do not exist yet, but it doesn't consider trying to map these connections to slots that don't exist yet. 

This is an issue. Ideally we would like "not currently existing" slots to be considered as potential slots we could map slot connections to. The plan is to leave it broken and revisit this issue later.

Reply:
maybe worth filing an issue for the revisiting / fixing this? 
my intuition is, Scott will start hitting these pretty soon, and it shouldn't be too hard to add support for (unless there is a reason not, which I am missing)

Reply:
Done :)